modules/widgets/rsslinks-widget.php

```diff
@@ -141,12 +141,19 @@ function _rss_link( $type = 'posts', $args ) {
 
 		$link_item = '';
 		$format = $args['format'];
+		
+		if ( has_filter( 'jetpack_rsslinks_widget_target_blank' ) && apply_filters( 'jetpack_rsslinks_widget_target_blank', false ) ) {
```

Comment:
Not sure why I did that. :confused:  Fix: https://github.com/Automattic/jetpack/pull/1776